monai/bundle/config_parser.py

```diff
@@ -116,6 +116,16 @@ def __init__(
     def __repr__(self):
         return f"{self.config}"
 
+    def __getattr__(self, id):
+        """
+        Get the config by id.
+
+        Args:
+            id: id to specify the expected position. See also :py:meth:`__getitem__`.
+
+        """
+        return self[id]
```

Comment:
have you tried returning `self.get_parsed_content("id")` here? maybe it's better to return the parsed content cc @Nic-Ma

Reply:
I agree with @wyli , this implementation doesn't match the initial feature request, we already have `dict` style `get / set` APIs to handle the config "text", this feature request is to **simplify the API of getting "parsed content"**, for example the instantiated objects ('dataloader', 'network', etc.) and also want to support nested items if the "parsed content" is still a `dict`.
Please check this recursive function for more details:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/config_parser.py#L359
Sorry for the confusion.

Thanks.

Reply:
Sorry, I fixed it, so it will now return the parsed content. It currently still only works one layer deep, so no nested items.